node-website-feed/README.md

```diff
@@ -11,3 +13,147 @@ Bare bones off the shelf website feed for NodeJS, including Client Credentials a
 5. Re-name the `src/.env.example` file to `.env` and add your `clientId` and `clientSecret`
 6. `yarn dev` or `npm run dev` to run a development server (`yarn start` or `npm run start` in prod)
 7. Load your browser at `http://localhost:3000`
+
+### Suggested Architecture
+
+In the code example, we fetch a list of properties on the fly. This is fine if you need the data to be the latest in our DB at the exact moment of execution however, practically for serving a website, it is expensive in terms of processing on the server, http round trips and financial cost by API calls.
+
+So how do I build on the website feed example?
+
+The simplest reasonable design pattern is as follows;
+
+#### 1. **Seed your application**
+
+Fetch all properties and embedded property images in a paged response, and commit to a database of your choosing. The query will be paginated at a maximum of 100 so you will need to do this step in a few requests, depending on your client.
+
+The following query will fetch all properties where;
+
+- The selling status is `forSale` - you can specify other statuses as you wish eg `underOffer`
+- The letting status is `toLet` and `currentlyAvailable` - as above, you can specify additional statuses
+- The customer has specified `internetAdvertising` is true
+- The page size is 100, and the page number is 1 - you will need to increment the page number until no more properties match your query
+- Embed property images. This will return all property images associated to each property. You should store these in a separate location in your database.
+
+```ts
+'https://platform.reapit.cloud/properties?sellingStatus=forSale&lettingStatus=toLet&lettingStatus=currentlyAvailable&internetAdvertising=true&embed=images&pageSize=100&pageNumber=1'
+```
+
+When you have seeded your DB, you can use this data to serve your website, removing the need to make a round trip for the properties from our APIs each time, thus keeping you costs to a minimum.
+
+#### 2. **On a schedule, fetch the properties that change**
+
+When you have your initial seed, you will need to update your data with any new or changed properties. This should be done with a scheduled job that queries both the properties and property images that change or are created, matching your filters.
+
+The following query will fetch the properties where;
+
+- The selling status, letting status, internet advertising and paging are as per the seed at stage 1
+- Images **are not** embedded in the property as it would fetch all images regardless of whether they change or not
+- The modified dates too and from match the last hour. This assumes that you will run your scheduled job every hour - you can adjust the timeframe based on how up to date you need your data to be however, more frequent fetching will result in greater cost.
```

Comment:
to not too

Reply:
Incidentally you don't really need to pass the modifiedTo parameter. You care about everything modified after your last import, so passing modifiedFrom is sufficient